src/fsharp/CompileOps.fs

```diff
@@ -2880,8 +2880,11 @@ type TcConfig private (data: TcConfigBuilder, validate: bool) =
           [ 
             // Check if we are given an explicit framework root - if so, use that
             match tcConfig.clrRoot with 
-            | Some x -> 
-                yield tcConfig.MakePathAbsolute x
+            | Some x ->
+                let clrRoot = tcConfig.MakePathAbsolute x
+                yield clrRoot
+                let clrFacades = Path.Combine(clrRoot, "Facades")
```

Comment:
Why is `tcConfig.clrRoot` not the right directory to begin with?
Does adding this heuristic for "Facades" mean it can happen on any directory that has a "Facades" folder?

Reply:
When mscorlib is not passed clrRoot is computed as the versioned directory in the reference assemblies directory. That will have a facades.  When mscorlib is passed in then it will either have a facades subdirectory if it was a deployed reference assembly or not.   Regardless, this just says also probe the Facades subdirectory for finding any reference assemblies which may be there.  Also, I think this shows up with the SimulatedMSBuildResolver, which is used by the IDE not the command line compiler.

Kevin